docs/c-ffi/calling-c.md

```diff
@@ -2,44 +2,33 @@
 
 FFI is built into Pony and native libraries may be directly referenced in Pony code. There is no need to code or configure bindings, wrappers or interfaces.
 
-Here's an example of an FFI call in Pony from the standard library. It looks like a normal method call, with just a few differences:
-
-```pony
-@fwrite[U64](data.cstring(), U64(1), data.size(), _handle)
-```
-
-The main difference is the @ symbol before the function name. This is what tells us it's an FFI call. Any time you see an @ in Pony there's an FFI going on.
-
-The other key difference is that the return type of the function is specified after the function name, in square brackets. This is because the compiler needs to know what type the value returned is (if any), but has no way to determine that, so it needs you to explicitly tell it.
-
-There are a few unusual things going on with the arguments to this FFI call as well. For the second argument, for which we're passing the value 1, we've had to specify that this is a U64. Again this is because the compiler needs to know what size argument to use, but has no way to determine this.
-
```

Comment:
whats the reason for the removal of this? At least out of context, this seems like important information.

Reply:
My rationale is that by removing the syntax for writing out the return type of a FFI function at the call site and requiring a declaration, it's hard to show a one-line example, so I decided to skip the intro and start the chapter by showing how to write a declaration.

The paragraph about having to type `U64(1)` is not needed anymore, since the compiler can now infer the correct type by looking at the declaration. This means one is allowed to write `1` as the second argument without any problems.

The bit about the @ symbol is covered a bit later.

Reply:
Ok, I am ok with this. We can see if people have any issues with it.